jsplot/src/main/java/tech/tablesaw/plotly/components/Axis.java

```diff
@@ -350,6 +354,7 @@ public String asJavascript() {
         if(showLine != DEFAULT_SHOW_LINE) context.put("showLine", showLine);
         if(zeroLine != DEFAULT_ZERO_LINE) context.put("zeroLine", zeroLine);
         if(showGrid != DEFAULT_SHOW_GRID) context.put("showGrid", showGrid);
+        if(domain != null) context.put("domain", String.format("[%.2f, %.2f]", domain[0], domain[1]));
```

Comment:
ben and I roughly follow Google coding conventions so (minor) please leave a space between if and the open paren, e.g if ( and not if(.

similarly but a bit more important, please always wrap the body of the if statement in parens.

Reply:
I made the changes as requested, just want to note that existing lines above it do not follow the convention requested.